packages/grpcweb-transport/src/grpc-web-transport.ts

```diff
@@ -70,12 +70,22 @@ export class GrpcWebFetchTransport implements RpcTransport {
     }
 
 
-    clientStreaming<I extends object, O extends object>(/*method: MethodInfo<I, O>, options: RpcOptions*/): ClientStreamingCall<I, O> {
-        throw new RpcError('Client streaming is not supported by grpc-web', GrpcStatusCode[GrpcStatusCode.UNIMPLEMENTED]);
+    clientStreaming<I extends object, O extends object>(method?: MethodInfo<I, O>/*, options: RpcOptions*/): ClientStreamingCall<I, O> {
+        const e = new RpcError('Client streaming is not supported by grpc-web', GrpcStatusCode[GrpcStatusCode.UNIMPLEMENTED]);
+        if (method) {
+            e.methodName = method.name;
+            e.serviceName  = method.service.typeName;
+        }
+        throw e;
     }
 
-    duplex<I extends object, O extends object>(/*method: MethodInfo<I, O>, options: RpcOptions*/): DuplexStreamingCall<I, O> {
-        throw new RpcError('Duplex streaming is not supported by grpc-web', GrpcStatusCode[GrpcStatusCode.UNIMPLEMENTED]);
+    duplex<I extends object, O extends object>(method?: MethodInfo<I, O>/*, options: RpcOptions*/): DuplexStreamingCall<I, O> {
+        const e = new RpcError('Duplex streaming is not supported by grpc-web', GrpcStatusCode[GrpcStatusCode.UNIMPLEMENTED]);
+        if (method) {
+            e.methodName = method.name;
+            e.serviceName  = method.service.typeName;
+        }
+        throw e;
```

Comment:
Same here - safe to require the argument.

```suggestion
    duplex<I extends object, O extends object>(method: MethodInfo<I, O>/*, options: RpcOptions*/): DuplexStreamingCall<I, O> {
        const e = new RpcError('Duplex streaming is not supported by grpc-web', GrpcStatusCode[GrpcStatusCode.UNIMPLEMENTED]);
        e.methodName = method.name;
        e.serviceName  = method.service.typeName;
        throw e;
```